WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -218,18 +268,37 @@ private static MockRpcClient CreateAndConfigureRpcClient(Func<int, FeeRate> esti
 					}),
 				OnGetPeersInfoAsync = async () =>
 					await Task.FromResult(hasPeersInfo ? new[] { new PeerInfo() } : Array.Empty<PeerInfo>()),
-
-				OnGetMempoolInfoAsync = async () =>
-					await Task.FromResult(new MemPoolInfo
-					{
-						MemPoolMinFee = memPoolMinFee // 1 s/b (default value)
-					}),
+				OnGetMempoolInfoAsync = async () => await Task.FromResult(ParseMempoolInfo(mempoolMinFee)),
 				OnEstimateSmartFeeAsync = async (target, _) =>
 					await Task.FromResult(new EstimateSmartFeeResponse
 					{
 						Blocks = target,
 						FeeRate = estimator(target)
 					})
 			};
+
+		private static MemPoolInfo ParseMempoolInfo(double mempoolMinFee)
+		{
+			var mempoolInfoWithHistogram = File.ReadAllText("./UnitTests/Data/MempoolInfoWithHistogram.json");
+			var jo = JObject.Parse(mempoolInfoWithHistogram);
+			var feeHistogram = (JObject)jo["fee_histogram"];
+
+			return new MemPoolInfo()
+			{
+				MemPoolMinFee = mempoolMinFee,
+				Histogram = feeHistogram.Properties()
+					.Where(p => p.Name != "total_fees")
+					.Select( p => new FeeRateGroup
+					{
+						Group = int.Parse(p.Name),
+						Sizes = p.Value.Value<ulong>("sizes"),
+						Count = p.Value.Value<uint>("count"),
+						Fees = Money.Satoshis(p.Value.Value<ulong>("fees")),
+						From = new FeeRate(p.Value.Value<decimal>("from_feerate")),
+						To = new FeeRate(Math.Min(100_000, p.Value.Value<decimal>("to_feerate")))
+					})
+					.ToArray()
+			};
+		}
 	}
-}
+}
```

Comment:
@lontivero please set your formatter to have a new line at the end of the files or at least to not remove.